jerry-core/ecma/builtin-objects/typedarray/ecma-builtin-typedarray-prototype.c

```diff
@@ -798,68 +798,60 @@ ecma_builtin_typedarray_prototype_join (ecma_object_t *obj_p, /**< this object *
     return ECMA_VALUE_ERROR;
   }
 
-  ecma_value_t ret_value = ECMA_VALUE_ERROR;
-
   /* 3. */
   uint32_t length = ecma_number_to_uint32 (length_number);
```

Comment:
If I understand the code correctly we could also free the `length_value` here. After this point it is only released and not used in any other ways.